cookbooks/kinematically_driven_subduction_2d/kinematically_driven_subduction_2d_case2a.prm

```diff
@@ -13,9 +13,10 @@
 # between the subducting and overriding plate, so about
 # 100 MPI processes are required.
 
-set Additional shared libraries            = /scratch/usr/bbpanneg/software/aspect/QQ_subduction_cookbook/shared_libs/build_release/lib2d_driven_subduction_box.so
+# We included a shared library for using the initial temperature plugin and the isotherm depth, composition RMS temperature, trench location postprocessors.
+set Additional shared libraries            = /home/bbkyingy/software/aspect_dealii_9.4.0/aspect/cookbooks/kinematically_driven_subduction_2d/libsubduction_plate_cooling.so
 
-# 2D
+# 2D 
```

Comment:
It seems like spaces have been added at the end of multiple lines, can you undo that?

Reply:
Still some spaces, see line 41 and 44